docs/utils.md

```diff
@@ -159,3 +159,5 @@ magic.util.uid(24)
 ### magic.alt.util.sha{256,512}
 
 Implements `SHA2` using OpenSSL through `crypto`. Each of the `SHA256`, `SHA384` (as `magic.util.hash`), and `SHA512` digest length variatns are available. An alterative to `magic.util.hash`.
+
```

Comment:
The line above this one has:

- `variatns`, should be `variants`
- `alterative`, should be `alternative`